.github/workflows/on-remove.yml

```diff
@@ -163,6 +163,7 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-default-region: ${{ steps.setup.outputs.aws-default-region }}
       - name: Delete the index
+        if: ${{ steps.setup.outputs.crn-algolia-index != 'asap-hub_dev' }}
```

Comment:
isn't this a little too specific, shouldn't we rely on the environment or label or something instead? like if we ever change the naming convention it would be good if this still worked as expected.

Plus - if you really are going to be excluding the dev index you might as well add prod in there too just to be safe